src/components/transactions/Repay/RepayActions.tsx

```diff
@@ -34,15 +31,17 @@ export const RepayActions = ({
   blocked,
   ...props
 }: RepayActionProps) => {
-  const { currentChainId: chainId, currentMarketData } = useProtocolDataContext();
-  const repay = useRootStore((state) => state.repay);
-  const repayWithPermit = useRootStore((state) => state.repayWithPermit);
+  const [repay, repayWithPermit, tryPermit] = useRootStore((state) => [
```

Comment:
This could be simplified to `const { repay, repayWithPermit, tryPermit } = useRootStore()`?

Reply:
Done